### DISCUSSION TITLE
Add copy and open explorer icons to transaction detail

### DISCUSSION THREAD

**Discussion by Phyloiota** [iotaledger/firefly#1578] [Features / Enhancements]
## Description

In the moment the detail view of a message offers copying to clipboard while you hover over the message id, sent address, receive address. Change this and use icons instead. 1 icon that copies and 1 icon that directly uses the content and opens explorer.iota.org. Also implement a success popup message "copied to clipboard". And maybe add tooltips if you hover over the icons (copy to clipboard, view in explorer)

## Motivation

atm its not obvious for the user that the option to copy single details exists. Icons will make it easier and clearer. Also the lookup option in explorer is not yet implemented

## Requirements
in this view
<img width="275" alt="Screenshot 2021-03-25 081801" src="https://user-images.githubusercontent.com/77154511/112400541-880fe380-8d43-11eb-82a5-b9ba71f6bbac.png">

add icons next to message, send address, receive address like this:
<img width="434" alt="Screenshot 2021-03-25 081240" src="https://user-images.githubusercontent.com/77154511/112400520-7c242180-8d43-11eb-9d1a-8fc770b7f1db.png">

## Open questions (optional)

Use this section to ask any questions that are related to the feature.

## Are you planning to do it yourself in a pull request?

no

**Comment by UmbahaSpodPaha** [comment]
Just came up with the idea that it would be great to have explorer link in transaction details and found this discussion.

Soooo +1 on this from me, please add at least Explorer link.
